### PR TITLE
Follow-up edits to PR#2575

### PR DESCRIPTION
**Pull request by ahardin-rh**
Follow-up to https://github.com/openshift/openshift-docs/pull/2575

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@adellape @tpoitras @bfallonf PTAL. Thanks!

File: install_config/storage_examples/binding_pv_by_label.adoc [hunk 1]
```diff
@@ -140,10 +145,11 @@ creating our objects.
 ----
 ====
 
-=== Deploy the pv, pvc, and endpoints
+[[binding-pv-by-label-deploy-pv-pvc-and-endpoints]]
+=== Deploy the PV, PVC, and Endpoints
 
-For this example, we will be running `oc` commands as a *cluster-admin*
-privileged user.  In a production environment, cluster clients might be expected
+For this example, we will run `oc` commands as a *cluster-admin*
```
Comment (adellape) [inline install_config/storage_examples/binding_pv_by_label.adoc]: s/we will run/run the/

File: install_config/storage_examples/binding_pv_by_label.adoc [hunk 2]
```diff
@@ -14,35 +14,38 @@ toc::[]
...
-available for all
-xref:../persistent_storage/index.adoc#[storage options].  It is assumed that
-a {product-title} cluster is running a storage volume is available for binding.
-
-===== _A note on Labels and Selectors_
+xref:../../architecture/additional_concepts/storage.adoc#persistent-volume-claims[persistent
+volume claims] (PVCs) to
+xref:../../architecture/additional_concepts/storage.adoc#persistent-volumes[persistent
+volumes] (PVs) by defining labels in the PV and matching selectors in the PVC.
+This feature is available for all xref:../persistent_storage/index.adoc#[storage
```
Comment (adellape) [inline install_config/storage_examples/binding_pv_by_label.adoc]: Needs ID after `#`.

File: install_config/storage_examples/binding_pv_by_label.adoc [hunk 3]
```diff
@@ -14,35 +14,38 @@ toc::[]
...
 labels can then be targeted by selectors to access all objects with matching
 label values. It is this functionality we will take advantage of to enable our
-PVC to bind to our PV.  For a more in-depth look at labels, refer to
+PVC to bind to our PV. For a more in-depth look at labels, refer to
 xref:../../architecture/core_concepts/pods_and_services.adoc#labels[Pods and Services].
 
 [NOTE]
 ====
 For this example, we will be using modified
 xref:../persistent_storage/persistent_storage_glusterfs.adoc#[GlusterFS]
```
Comment (adellape) [inline install_config/storage_examples/binding_pv_by_label.adoc]: Needs ID after `#`.

File: install_config/storage_examples/binding_pv_by_label.adoc [hunk 4]
```diff
@@ -14,35 +14,38 @@ toc::[]
...
-PV and PVC specifications.  However, implementation of selectors and labels is
- the same for all storage options.  Refer to the
+PV and PVC specifications. However, implementation of selectors and labels is
+ the same for all storage options. Refer to the
  xref:../persistent_storage/index.adoc#[relevant storage option] for your volume
```
Comment (adellape) [inline install_config/storage_examples/binding_pv_by_label.adoc]: Needs ID after `#`.

File: install_config/storage_examples/binding_pv_by_label.adoc [hunk 5]
```diff
@@ -14,35 +14,38 @@ toc::[]
...
 provider to learn more about its unique configuration.
 ====
 
 [[binding-pv-by-label-assumptions]]
 === Assumptions
+It is assumed that you have:
+
 * An existing {product-title} cluster with at least one *master* and one *node*
 * At least one supported  xref:../persistent_storage/index.adoc#[storage volume]
```
Comment (adellape) [inline install_config/storage_examples/binding_pv_by_label.adoc]: Needs ID after `#`.

**Comment by adellape** [comment]
@ahardin-rh Comments inline, otherwise LGTM. There was a hanging `#` that I couldn't actually comment on; should be 5 total hits in `binding_pv_by_label.adoc` needing them.

File: install_config/storage_examples/binding_pv_by_label.adoc [hunk 6]
```diff
@@ -14,37 +14,41 @@ toc::[]
...
+
 Labels are an {product-title} feature that allow for user defined tags to be
-included with object specifications.  Their primary purpose is to enable the
-arbitrary grouping of objects by defining identical labels among them.  These
+included with object specifications. Their primary purpose is to enable the
+arbitrary grouping of objects by defining identical labels among them. These
 labels can then be targeted by selectors to access all objects with matching
 label values. It is this functionality we will take advantage of to enable our
-PVC to bind to our PV.  For a more in-depth look at labels, refer to
+PVC to bind to our PV. For a more in-depth look at labels, refer to
```
Comment (bfallonf) [inline install_config/storage_examples/binding_pv_by_label.adoc]: s/refer to/see

**Comment by bfallonf** [comment]
@ahardin-rh A few references to "refer to" instead of "see", but otherwise LGTM!

**Comment by ahardin-rh** [comment]
@adellape @bfallonf thanks for the peer review!

**Comment by ahardin-rh** [comment]
Rev history in original PR

**Pull request merged**